news/2020-04-07-mappers-report-march.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title: "The Mappers' Report: February"
+date: 2020-04-10 20:00:00 +0000
+---
+
+Another month passed and that means it’s time for another Mappers’ Report! Full of news about the Ranking Criteria, meetings and the Beatmap Nominators! Additionally, this month we are ready to introduce the Mappers’ Showcase as well as other community news! 
+So let’s jump right into it!
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+This month's issue of the Mappers' Report contains following topics:
+
+- **[Ranking Criteria Changes](#rc-changes)**
+- **[NAT Meetings](#nat-meetings)**
+- **[Within the Beatmap Nominators and NAT](#within-bn-nat)**
+- **[Mappers’ Showcase](#mappers-showcase)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Around the Community](#around-community)**
+- **[Conclusion](#conclusion)**
+
+## <a id="rc-changes"></a>Ranking Criteria Changes
+
+The Ranking Criteria. For one a blessing, for others a curse. And it’s changing all the time. This month we didn’t have many changes in the Ranking Criteria, that’s why we had a great idea. From now on we will also list ongoing discussions in the Mappers’ Report to give them a bit more of attention. The Ranking Criteria affects *all* mappers, and everyone should be aware of what is going on or what is going to happen. So let’s start right away with the changes we had this month. 
+
+### All modes
+
+- **[Applause File and Passive Hitsounds](https://osu.ppy.sh/help/wiki/Ranking_Criteria#guidelines.4):** Previously a Rule, this got now moved to the Guidelines. Applause files and passive hitsounds should use .mp3 or .ogg files. The only exception of this is when using a .wav file will result in smaller file size.
+
+- **[Generalised Mode-Specific Guidelines about Custom Hitsounds](https://osu.ppy.sh/help/wiki/Ranking_Criteria#guidelines.4):** Previously split between the General and osu!catch Ranking Criteria, this guideline exist only in the general Ranking Criteria now. The hit finish in soft and normal sample sets should not be replaced with frequently used hitsounds because these can sound obnoxious for anyone disabling beatmap hitsounds. osu!taiko has an exception of this Guideline because of own mode specific hitsound sample guideline. 
+
+### osu!taiko 
+
+- **[1/3 Pattern in Futsuu](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!taiko#-futsuu):** A change to the Rule and Guideline regarding 1/3 pattern was made. The clause about the rest moment was moved to the Guideline. 
+
+## Open Ranking Criteria Proposals 
+
+As said previously, the Ranking Criteria is a never ending process. Everyone can suggest changes to it and there are currently a lot of proposals which need more input from the community! Every proposal can be found in the [Ranking Criteria Forum](https://osu.ppy.sh/community/forums/87). Here a small list of the most important and longest open one:
+
+For all modes:
```

Comment:
Add https://osu.ppy.sh/community/forums/topics/1044301 to this all modes section?